packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/useHandleUpload.ts

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
-import { uploadData, UploadDataWithPathInput } from 'aws-amplify/storage';
+import {
+  uploadData,
+  UploadDataWithPathInput,
+  isCancelError,
+} from 'aws-amplify/storage';
 
 import { useGetLocationConfig } from '../../context/config';
 import { TaskStatus } from '../../context/types';
 
+// 5MB for multipart upload
+export const MULTIPART_UPLOAD_THRESHOLD_BYTES = 5242880;
```

Comment:
does this exact number match the one used by the JS library? If so, can we include a link?

Reply:
I didn't get it from the JS team, but now that you mention it I should do that. I just looked it up on their repo and they make it a lot easier to understand: https://github.com/aws-amplify/amplify-js/blob/1a5366d113c9af4ce994168653df3aadb142c581/packages/storage/src/providers/s3/utils/constants.ts#L17

I'll add this as a const somewhere

Reply:
cool, and might as well link to the JS repo as well, since they are the source of truth for when we can cancel or not.